### PR TITLE
📜Remove hardcoded dotnet version reference

### LINKED ISSUE #320: 📜Remove hardcoded dotnet version reference
### Complete The Item Below

- [x] I have updated the title without removing the 📜 emoji.

### Description

Following line is currently out of date based on the change in https://github.com/KinsonDigital/Velaptor/commit/1ce17e9dd55a900a1c263eaba11d627d90853f28

```suggestion
- Create a new C# console application using dotnet _**v7.0**`
+ Create a new C# console application using dotnet version specified in the EmptyProject.csproj
```

### Acceptance Criteria

```[tasklist]
- [x] Documentation updated to: Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`
```

### ToDo Items

```[tasklist]
- [x] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [x] A priority label was added to this issue.  Refer to the _**Priority Type Labels**_ section below.
```

### Additional Information:

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label                                                                  |
|---------------------|------------------------------------------------------------------------|
| Low Priority        | https://github.com/KinsonDigital/VelaptorDocs/labels/low-priority    |
| Medium Priority     | https://github.com/KinsonDigital/VelaptorDocs/labels/medium-priority |
| High Priority       | https://github.com/KinsonDigital/VelaptorDocs/labels/high-priority   |
</details>

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct.

### PR DESCRIPTION
**Pull request by LeaDevelop**
### **Description:**

To allow this pull request to be merged, please make sure that the following items of this pull request are complete.

This pull request closes #320

✅The pull request head branch must be a feature branch with the syntax 'feature/<issue-num>-sync-testing'.  
✅The pull request base branch must be a 'main' or 'preview' branch..  
✅The pull request head branch contains a valid issue number.  
✅The pull request title matches the linked issue title exactly.  
✅The pull request assignees match the assignees of the issue.  
✅The pull request labels match the labels of the issue.  
✅The pull request organizational projects match the organizational projects of the issue.  
✅The pull requests milestone matches the milestone of the issue.  

---

### **Additional Info:**
1. Pull requests are automatically synced with the associated issue upon creation by one of the KinsonDigital workflow bots. 
2. The associated issue is the issue number that is embedded in the pull request head branch.
3. The list above will be automatically updated as the pull request's various settings match or do not match the associated issue.
4. To manually sync the pull request to the issue, create a comment with the `[run-sync]` command.
	> **Note** You must be an admin member of the organization to use this command.

---

- [x] Sync with the issue.  Use this to enable or disable this pull request from syncing with its associated issue.

### CODE REVIEW

File: docs/guides/guides/project-setup.mdx [hunk 1]
```diff
@@ -24,7 +24,7 @@ First things first. You have to create a project with your preferred IDE!!
 
 ## <span className="color-step">Step 1: Create the application</span>
 
-Create a new C# console application using dotnet _**v7.0**_
+Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look after property group `<TargetFramework>`.
```
Comment (CalvinWilkinson) [inline docs/guides/guides/project-setup.mdx]: Please make the following changes:

```suggestion
Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look for the `<TargetFramework></TargetFramework>` XML tag.
```

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

File: docs/guides/guides/project-setup.mdx (hunk 1 above)
Comment (LeaDevelop) [reply docs/guides/guides/project-setup.mdx]: That makes sense, good point out XML tag also newbie friendly 🌟 I should have put that there in the beginning.

**Comment by LeaDevelop** [comment]
@CalvinWilkinson any other corrections, or is this version fine now?

**Review by CalvinWilkinson — DISMISSED** [review]

File: docs/guides/guides/project-setup.mdx (hunk 1 above)
Comment (CalvinWilkinson) [reply docs/guides/guides/project-setup.mdx]: Np at all.

**Comment by CalvinWilkinson** [comment]
> @CalvinWilkinson any other corrections, or is this version fine now?

I think that is perfect.  The only other thing is that we should make the exact same change to the rest of the versions of the docs.

That would be these other versions shown below:
image

I should of had that in the issue but I did not think anybody was going to pick this up but me.  Sorry about that!

**Comment by LeaDevelop** [comment]
No worries, thanks for sharing where exactly changes should be I'll take care of it now

**Review by CalvinWilkinson — APPROVED** [review]

**Pull request merged**